pkg/apis/aquasecurity/v1alpha1/compliance_types.go

```diff
@@ -74,3 +75,10 @@ type ControlCheck struct {
 	FailTotal   int      `json:"failTotal"`
 	Severity    Severity `json:"severity"`
 }
+
+type DefaultStatus string
```

Comment:
See my comment about using boolean instead of enum. If you want to proceed with enum then we can may call it ControlStatus? And then use it as type of the DefaultStatus property.